scripts/bump-version.sh

```
@@ -0,0 +1,149 @@
+# Define target files
+FLUTTER_APP_USER="apps/app_user/pubspec.yaml"
+FLUTTER_APP_PARTNER="apps/app_partner/pubspec.yaml"
+SHARED_MINGLIT_KIT="shared/packages/minglit_kit/pubspec.yaml"
+SHARED_MINGLIT_LINTS="shared/packages/minglit_lints/pubspec.yaml"
+SHARED_MINGLIT_IAMPORT="shared/packages/minglit_iamport_v1/pubspec.yaml"
+NPM_LANDING_USER="apps/landing_user/package.json"
+NPM_LANDING_PARTNER="apps/landing_partner/package.json"
+NPM_ROOT="package.json"
+
+# Array of files to update
+declare -a FILES=(
```

Shellcheck (0.11.0)

[warning] FILES appears unused. Verify use (or export if used externally).

(SC2034)